releasenotes/notes/Added-random-circuit-from-graph-95c22eeabdea89d0.yaml

```diff
@@ -0,0 +1,71 @@
+---
+features_circuits:
+  - |
+    The function :func:`~qiskit.circuit.random.utils.random_circuit_from_graph` generates a random circuit that 
+    induces the same interaction graph as the interaction graph specified by `interaction_graph`.
+
+    The probability of randomly drawing an edge from the interaction graph as a two-qubit gate can be set by the 
+    user in the weight attribute of an edge in the input interaction graph. If the user does not set the probability, 
+    each edge is drawn uniformly, i.e. each two-qubit gate represented by an edge in the interaction graph has the 
+    same probability of getting added to the random circuit. If only a subset of edge probabilities are set, 
+    `ValueError` will be raised.
+
+    Arguments accepted by the function are
```

Comment:
I would remove the arguments paragraph, seems too much for a release note (users can go to the docstring itself for more details).

Reply:
Right, done !